<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashDialog</class>
 <widget class="QDialog" name="CashDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cash out</string>
  </property>
  <widget class="QPushButton" name="pb_cash">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cash out</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QDateEdit" name="dateEdit_1"/>
    </item>
    <item row="1" column="0">
     <widget class="QDateEdit" name="dateEdit_2"/>
    </item>
    <item row="2" column="0">
     <widget class="QComboBox" name="line_master"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
